### DISCUSSION TITLE
Student Account not proved

### DISCUSSION THREAD

**Discussion by usamariaz115** [community/community#51376] [GitHub Education]
I am trying to create my student account but always giving this error:

The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote.

You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so. 

I am living in the mentioned university hostel but even then its giving this error.

**Comment by samus-aran** [comment]
Hi @usamariaz115 👋 

Thanks for bringing this to our attention. We will pass this along to our internal team to investigate and any updates will be posted here. 👍

**Reply by samus-aran** [reply]
Hi @usamariaz115 

We have updated your selected school details. We please ask you to try and re-apply again and let us know how you get along 👍

**Reply by Tndot** [reply]
Hi @samus-aran 
I have the same problem. Could you help me update my school details? Thank you!
